bot/exts/pride/prideleader.py

```diff
@@ -0,0 +1,88 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+from fuzzywuzzy import fuzz
+
+from bot.constants import Colours
+log = logging.getLogger(__name__)
+
+
+class PrideLeader(commands.Cog):
+    """Gives information about Pride Leaders."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.pride = self.load_json()
+
+    @staticmethod
+    def load_json() -> dict:
+        """Loads pride leader information from static json resource."""
+        explanation_file = Path("bot/resources/pride/prideleader.json")
+        with explanation_file.open(encoding="utf8") as json_data:
+            pride = json.load(json_data)
+
+        return pride
+
+    def name_verifier(self, leader_name: str) -> str:
+        """Verify leader name wether it is present in json or not."""
+        leader_name = leader_name.split(" ")
+        leader_name = ' '.join(name.capitalize() for name in leader_name)
```

Comment:
use the title() method instead
```suggestion
        leader_name = leader_name.title()
```